plugins/chat/spec/system/thread_list/full_page_spec.rb

```diff
@@ -140,7 +130,7 @@
 
     it "opens a thread" do
       chat_page.visit_channel(channel)
-      channel_page.open_thread_list
```

Comment:
A lot of spec changes are related to the fact that we might now have the list opened when we navigate to a channel, so we don't have to open it.